tests/Event/Subscriber/TablePrefixSubscriberTest.php

```diff
@@ -115,6 +116,6 @@ public function testPrefixManyToMany()
 
         $this->assertSame('yo_entry', $metaDataEvent->getClassMetadata()->getTableName());
         $this->assertSame('yo_entry_tag', $metaDataEvent->getClassMetadata()->associationMappings['tags']['joinTable']['name']);
-        $this->assertSame('yo_entry', $metaDataEvent->getClassMetadata()->getQuotedTableName(new MySQLPlatform()));
+        $this->assertSame('yo_entry', (new DefaultQuoteStrategy())->getTableName($metaClass, new MySQLPlatform()));
```

Comment:
What if we switch to an other quote strategy in the entity?

Reply:
I did it this simple way because we're not changing the quote strategy in the config, and to keep it a unit test, otherwise it would have been an integration test with KernelTestCase at least